cypress-tests/cypress/e2e/PaymentTest/00028-MemoryCacheConfigs.cy.js

```diff
@@ -0,0 +1,35 @@
+import State from "../../utils/State";
+
+let globalState;
+
+describe("In Memory Cache Test", () => {
+  before("seed global state", () => {
+    cy.task("getGlobalState").then((state) => {
+      globalState = new State(state);
+    });
+  });
+
+  after("flush global state", () => {
+    cy.task("setGlobalState", globalState.data);
+  });
+
+  context("Config flows", () => {
+    const key = "test-key";
+    const value = "test value";
+    const newValue = "new test value";
```

Comment:
can these be made dynamic? you can use [`generateRandomString`](https://github.com/juspay/hyperswitch/blob/8092c1fd8fbe115ce7b3f4329758800e506bccae/cypress-tests/cypress/utils/RequestBodyUtils.js#L13) function from `RequestBodyUtils`.

Asking so because, i believe, `newValue` can be updated only once unless a different value is passed and this will cause errors when tests are run back to back without creating another merchant.

Reply:
As part of the test, we're creating a config first, updating it and finally deleting it. So, if the test runs successfully, the config should not exist and the next run should not be a concern.

That said, since these configs are NOT associated with any merchant or profile (but have a "global" scope), if there are two Cypress processes running this test case simultaneously, then the test case _could_ fail because both test cases would read and write to the same config key.

If we're running this test case for every connector and there's a possibility that we'd run tests for connectors in parallel, then we should either consider moving this test case out of connector-specific test cases, or consider suffixing a random string to reduce the possibility of collisions. I'm personally leaning towards the former, since having tests that are not associated with a merchant or a profile is a valid use case.